### ISSUE TITLE
Make modals and elements high contrast

### ISSUE THREAD

**Issue by raisedadead** [freeCodeCamp/freeCodeCamp#36691]
![image](https://user-images.githubusercontent.com/1884376/63763613-b2d55400-c8e2-11e9-8aa1-7dc8f81be390.png)

There is no distinction between elements like the modal and the background. The effect is best reproduced on a non-retina screen.

**Comment by ahmadabdolsaheb** [comment]
set 
```
 .modal-backdrop.in {
    opacity: 0.8;
}
```
in the end of client/src/components/layouts/global.css

**Comment by ahmadabdolsaheb** [comment]
@raisedadead, would darkening the backdrop suffice?
or would you like to have a clear distinction like a border?

![image](https://user-images.githubusercontent.com/4591597/64059779-a47d8580-cbcb-11e9-8f14-810668dcf31c.png)

![image](https://user-images.githubusercontent.com/4591597/64059777-99c2f080-cbcb-11e9-8d4e-6fe4956d0d6a.png)

**Comment by raisedadead** [comment]
> would darkening the backdrop suffice?

IMHO, Shades of similar color are bad for a11y. We need high contrast elements that visually differentiate between elements.

A 1px border of a contrast color should be better.

**Comment by raisedadead** [comment]
Note that you have to check this on a non-retina like screen to understand the side-effect on low quality screens. :(

**Comment by ahmadabdolsaheb** [comment]
I know this is radical. but how about a light backdrop for dark mode.?
![image](https://user-images.githubusercontent.com/4591597/64175287-a9f4fd00-ce63-11e9-86a9-ae57e2797660.png)

**Comment by ahmadabdolsaheb** [comment]
clearly no a good idea. a pr with border is coming up :)
